tests/unit/style/properties/serialization.rs

```diff
@@ -617,10 +657,9 @@ mod shorthand_serialization {
             properties.push(PropertyDeclaration::LineHeight(line_height));
 
             let serialization = shorthand_properties_to_string(properties);
-            assert_eq!(serialization, "font:;");
+            assert_eq!(serialization, "font: italic normal bolder expanded 4px/3 serif;");
```

Comment:
Also this would not pass, because shorthand requires all its subprops to be present in the declaration block, including those it cannot represent.

You can take the test code of `transition` at the end of this file as an example of how to simplify this test code.